shell/platform/darwin/macos/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -0,0 +1,71 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/darwin/macos/framework/Headers/FlutterEngine.h"
+#import "flutter/shell/platform/darwin/macos/framework/Headers/FlutterViewController.h"
+#include "flutter/shell/platform/darwin/macos/framework/Source/FlutterDartProject_Internal.h"
+#include "flutter/shell/platform/darwin/macos/framework/Source/FlutterEngine_Internal.h"
+#include "flutter/testing/testing.h"
+
+namespace flutter::testing {
+
+TEST(FlutterViewControllerTest, MacOSTestTest) {
+  NSString* fixtures = @(testing::GetFixturesPath());
+  FlutterDartProject* project = [[FlutterDartProject alloc]
+      initWithAssetsPath:fixtures
+             ICUDataPath:[fixtures stringByAppendingString:@"/icudtl.dat"]];
+  FlutterViewController* viewController = [[FlutterViewController alloc] initWithProject:project];
+
+  // First call setClipboardData to put a string on the pasteboard.
+  __block bool calledSet = false;
+  FlutterResult resultSet = ^(id result) {
+    calledSet = true;
+  };
+  FlutterMethodCall* methodCallSet =
+      [FlutterMethodCall methodCallWithMethodName:@"Clipboard.setData"
+                                        arguments:@{@"text" : @"some string"}];
+  [viewController handleMethodCall:methodCallSet result:resultSet];
+  ASSERT_TRUE(calledSet);
+
+  // Call hasStrings and expect it to be true.
+  __block bool called = false;
+  __block bool value;
+  FlutterResult result = ^(id result) {
+    called = true;
+    NSNumber* valueNumber = [result valueForKey:@"value"];
+    value = [valueNumber boolValue];
+  };
+  FlutterMethodCall* methodCall =
+      [FlutterMethodCall methodCallWithMethodName:@"Clipboard.hasStrings" arguments:nil];
+  [viewController handleMethodCall:methodCall result:result];
+  ASSERT_TRUE(called);
+  ASSERT_TRUE(value);
```

Comment:
I suspect the unit tests on the bots are running in a bootstrap context that doesn't have access to the pasteboard server. If that's the case, perhaps the tests should inject a mock `NSPasteboard`.

Reply:
Any idea how to include OCMock in the Mac tests?  I tried including it in this test file and adding it to Build.gn in various places, but I keep getting compiler errors.

Reply:
The same way it's done for iOS? Without knowing what you tried and what exactly didn't work, it's hard to advise further.